### DISCUSSION TITLE
Should I create a PR to demonstrate a break in backward compatibility?

### DISCUSSION THREAD

**Discussion by benbankes** [TanStack/query#668] [General] [answered]
I would like to know how to proceed.

I have detailed a backward-compatibility break in https://github.com/tannerlinsley/react-query/issues/664.  I do not think I can use react-query 2.4.5 for my use case.  I am trying to create an optimistic UI, while also displaying any errors that the API sends back to the user.  I am able to do one or the other, but not both.

To demonstrate the BC break, I think I could quickly create a failing test that uses an example from the documentation, but I am unsure if that would be welcome as a PR.

What is the best way to proceed?

**Comment by tannerlinsley** [comment, marked as answer]
Start by making a codesandbox that shows how the error is reproduced and how you would expect it to work, then post that codesandbox in a new issue.
